UI/templates/question.html

```diff
@@ -0,0 +1,100 @@
+                    <li><a href="profile.html" class="header-right shake"><img src="../images/icon/user.svg" class="icon">Profile</a></li>
+                    <li><a href="admin.html" class="header-right shake"><img src="../images/icon/admin-with-cogwheels.svg" class="icon">admin</a></li>     
+                </ul>
+                
+            </nav>
+    </div>
+
+<h2>Machine Learning Workshops</h2>
+
+<details open>
+  <summary><img src="../images/avatar2.png" class="Avatar" style="width:90px">
+  	  	<p></span>Chris Fox. <span class="txt1">Posted 2 days ago</span></p>
+  		What is machine learning?</summary>
+          <div class="rating">
+            <i class="button rating-up fa fa-thumbs-o-up" aria-hidden="true"></i>
+            <i class="button rating-down fa fa-thumbs-o-down" aria-hidden="true"></i>
+            <span class="counter">30</span>
+        </div>
+          <ul>
+            <hr class="comments-line">
+            <li><h5>Comments</h5></li>
+            <hr class="comments-line">
+            <li><p>Totally. Totally does.</p></li>
+            <hr class="comments-line">
+            <li><p>Totally. Totally does.</p></li>
+            <hr class="comments-line">
+            <li><p>Totally. Totally does.</p></li>
+            </ul>
+</details>
+
+<details>
+    <summary><img src="../images/avatar_beanie_girl-512.png" class="Avatar" style="width:90px">
+        <p></span>Chris Fox. <span class="txt1">Posted 5 minutes ago</span></p>
+      where do I start from?</summary>
+      <div class="rating">
+		<i class="button rating-up fa fa-thumbs-o-up" aria-hidden="true"></i>
+		<i class="button rating-down fa fa-thumbs-o-down" aria-hidden="true"></i>
+		<span class="counter">23</span>
+	</div>
+      <ul>
+        <li><p>Totally. Totally does.</p></li>
+        <hr>
+        <li><p>Totally. Totally does.</p></li>
+        <hr>
+        <li><p>Totally. Totally does.</p></li>
+        <hr>
+        <li><p>Totally. Totally does.</p></li>
+        </ul>
+</details>
+
+<details>
+    <summary><img src="../images/Avatar_1.resized.png" class="Avatar" style="width:90px">
+        <p></span>Chris Fox. <span class="txt1">Posted 5 minutes ago</span></p>
+      Anyone suggest me the resources for machine learning?
+      
+    </summary>
+    <div class="rating">
+		<i class="button rating-up fa fa-thumbs-o-up" aria-hidden="true"></i>
+		<i class="button rating-down fa fa-thumbs-o-down" aria-hidden="true"></i>
+		<span class="counter">8</span>
+	</div>
+      <ul>
+        
+        <li><p>Comments</p></li>
+        <hr>
+        <li><p>Totally. Totally does.</p></li>
+        <hr>
+        <li><p>Totally. Totally does.</p></li>
+        <hr>
+        <li><p>Totally. Totally does.</p></li>
+        </ul>
+</details>
+</body>
+</html>
+
+
+<footer>
+    <p>Questioner, Copyright &copy; 2019</p>
+</footer>
+   
+</body>
+</html>
```

Comment:
needs a new line

Reply:
ah what do you mean by newline?

Reply:
You need a new line with no code at the end of your file.